tests/manual/util/README.md

```diff
@@ -0,0 +1,33 @@
+# Test Utilities
+
+## ManualTestWindow
+
+This facility provides a framework for manual tests to display test instructions, test pane, and Pass/Fail buttons.
+
+A simple test would look like this:
+
+```java
+public class SampleManualTest {
+     public static void main(String[] args) throws Exception {
+         ManualTestWindow.builder().
+             title("Sample Manual Test").
+             instructions(
+                 """
+                 Provide
+                 multi-line instructions here.
+                 """
+             ).
+             ui(() -> createTestUI()).
+             buildAndRun();
+     }
+
+     private static Node createTestUI() {
+         return new Label("Test UI");
+     }
+}
+```
+
+Resulting application window:
```

Comment:
Resulting application window does not look like it is from example above. Not sure if it is important.

Reply:
Not really important.

Apart from being resized to decrease the size of the PNG image, it looks exactly the same.  Are you running it on a different platform?  How does it look, anything appears missing (can you post a screenshot)?

Reply:
If you run `EmojiTest`, then yes it will look correct, but if you run `SampleManualTest` it will not. Since you added this image to `SampleManualTest` as resulting image of `SampleManualTest`. Look at [ReadMe.md](https://github.com/openjdk/jfx/blob/a106587a69859966e668233a9d89539c3438fed7/tests/manual/util/README.md).

Reply:
indeed!  thanks!